app/forms/openid_connect_authorize_form.rb

```diff
@@ -101,7 +95,7 @@ def link_identity_to_service_provider(
       nonce: nonce,
       rails_session_id: rails_session_id,
       ial: ial,
-      acr_values: acr_values&.join(' '),
+      acr_values: Vot::AcrComponentValues.join(acr_values),
```

Comment:
I think `.join` is a confusing name for a custom method here, it looks like that's parsing + reformatting the `acr_values`. Also, looks like `.join()` seems to support either an array or strings, but this is the only callsite and it should always be an array right?

I also think if we're going to use a model class it might be clear to have like separate parse from format steps, like:

```suggestion
      acr_values: Vot::AcrComponentValues.from(acr_values).to_s,
```

Reply:
That's a good callout. This is very much still a work in progress!